pydrive2/test/test_apiattr.py

```diff
@@ -1,34 +1,39 @@
 import unittest
+import pytest
 
 from pydrive2.auth import GoogleAuth
 from pydrive2.drive import GoogleDrive
+from pydrive2.test.test_util import pydrive_retry, setup_credentials
 
+
+@pytest.mark.automated
 class ApiAttributeTest(unittest.TestCase):
```

Comment:
why don't we use `GoogleDriveTest` here?

Reply:
or some other generic test that deals with the regular auth that is common for most of the test as far as I understand?

Reply:
> why don't we use GoogleDriveTest here?

original auth implementation is not common for all automated TestCase classes. For example, it might be:

```
@classmethod
    def setup_class(cls):
        setup_credentials()
```

or

```
@classmethod
    def setup_class(cls):
        setup_credentials()

        cls.ga = GoogleAuth('pydrive2/test/settings/default.yaml')
        cls.ga.ServiceAuth()
```

I've stopped refactoring there on common logic moving to the `setup_credentials()` defined in utils.

Reply:
👍